pkg/minikube/assets/addons.go

```diff
@@ -36,11 +36,13 @@ import (
 
 // HelmChart holds information about a helm chart.
 type HelmChart struct {
-	Name       string
-	Repo       string
-	Namespace  string
-	Values     []string
-	ValueFiles []string
+	Name        string
+	Repo        string
+	RepoURL     string
+	Namespace   string
+	Values      []string
+	ValueFiles  []string
+	ImageSetKey string
```

Comment:
@PhantomInTheWire 
why is this called ImageSetKey is that same Name used in helm flags? can you please put an example of using helm itself with image respostiry 

also add comment to this field